package.json

```diff
@@ -852,7 +852,7 @@
         "portfinder": "^1.0.13",
         "request": "^2.83.0",
         "request-promise": "^4.2.2",
-        "vscode-azureappservice": "^0.23.0",
+        "vscode-azureappservice": "^0.24.1",
```

Comment:
You need to also bump 'azure-arm-website' to '5.whatever'. We bumped it in the 0.24.0 release. See here: https://github.com/Microsoft/vscode-azuretools/blob/master/appservice/package.json#L37